roles/ami_config/tasks/ami-tower/install.yml

```diff
@@ -83,4 +83,29 @@
     password: "{{ tower_password }}"
     body: '{{ lookup("file", tower_license) }}'
     body_format: json
+
+- name: Enable sudo escalation for Tower
+  lineinfile:
```

Comment:
I will test tomorrow, but for now, is this idempotent? One thing I'd recommend is first doing a grep for the line you want and registering it, and only do lineinfile if the entry is not there.